.github/workflows/scripts/ping-codeowners-on-new-issue.sh

```diff
@@ -65,6 +65,7 @@ for COMPONENT in ${BODY_COMPONENTS}; do
     fi
 
     PING_LINES+="- ${COMPONENT}: ${CODEOWNERS}\n"
+    # shellcheck disable=SC2034
```

Comment:
When using `shellcheck disable`, can you add a comment explaining why you did it?

Reply:
AIUI, this is used in line 63 so I think we should change the comment to say that